Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -882,6 +882,12 @@
 (deny mach-lookup
     (global-name "com.apple.mobilegestalt.xpc"))
 
+(allow mach-lookup (with telemetry-backtrace)
```

Comment:
Do we need the `telemetry-backtrace` when we are launching? Don't we expect that to be happening?

Reply:
Ah, that is a good point! I have changed this to telemetry only, since we normally don't rely on this access. It is only needed when the cache is invalid.

Thanks for reviewing!